ionos_sbx/wallet/vault/templates/cronjob.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-check-cronjob
+  labels:
+    app: {{ .Release.Name }}
+spec:
+  schedule: "*/5 * * * *"  # Run every 5 minutes
+  jobTemplate:
+    spec:
+      ttlSecondsAfterFinished: 0  # Delete job immediately after completion
+      template:
+        spec:
+          serviceAccountName: vault-check-sa
+          containers:
+            - name: vault-check
+              image: bitnami/kubectl:latest
+              command: ["/bin/sh", "-c"]
+              args:
+                - |
+                      #!/bin/sh
```

Comment:
I am missing here quick check at the beginning that would just quit the job if vault is initiallized,sealed and unsealed. Can you add something like this?

Reply:
Done